### PR TITLE
doc(Issue:1905):Improve doc titles in order to better be returned in the searches

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description of the change:**
Improve doc titles across the repo in order them be returned in the searches

**Motivation for the change:**
Closes: https://github.com/operator-framework/operator-sdk/issues/1905

### CODE REVIEW

File: doc/proposals/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Proposals
+# Proposals for Operator SDK
```
Comment (joelanford) [inline doc/proposals/README.md]:
```suggestion
# Operator SDK Proposals
```

File: doc/sdk-cli-reference.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# CLI Guide
+# CLI Guide for Operator SDK
```
Comment (joelanford) [inline doc/sdk-cli-reference.md]:
```suggestion
# Operator SDK Command Line Interface (CLI) Reference
```

File: doc/test-framework/scorecard.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Operator Scorecard
+# Operator Scorecard feature from Operator SDK
```
Comment (joelanford) [inline doc/test-framework/scorecard.md]:
```suggestion
# Operator SDK Scorecard
```

File: doc/user-guide.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Operator SDK User Guide
+# User Guide for Operator SDK
```
Comment (joelanford) [inline doc/user-guide.md]: Revert. I think this is good as is:

```suggestion
# Operator SDK User Guide
```

File: doc/user/client.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# Operator-SDK: Controller Runtime Client API
+# Controller Runtime Client API with Operator SDK
```
Comment (joelanford) [inline doc/user/client.md]:
```suggestion
# Using the Controller Runtime Client API with Operator SDK
```

File: doc/user/event-filtering.md [hunk 6]
```diff
@@ -1,4 +1,4 @@
-# Event filtering with Predicates
+# Event filtering with Predicates with Operator SDK
```
Comment (joelanford) [inline doc/user/event-filtering.md]:
```suggestion
# Using Predicates for Event Filtering with Operator SDK
```

File: doc/user/logging.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
-# Logging in operators
+# Logging in operators with Operator SDK
```
Comment (joelanford) [inline doc/user/logging.md]:
```suggestion
# Operator SDK Logging
```

File: doc/user/metrics/README.md [hunk 8]
```diff
@@ -1,4 +1,4 @@
-# Monitoring with Prometheus
+# Monitoring with Prometheus for Operator SDK
```
Comment (joelanford) [inline doc/user/metrics/README.md]:
```suggestion
# Operator SDK Monitoring with Prometheus
```

File: doc/user/metrics/service-monitor.md [hunk 9]
```diff
@@ -1,4 +1,4 @@
-## Using the ServiceMonitor prometheus-operator CRD
+## Using the ServiceMonitor prometheus-operator CRD for Operator SDK
```
Comment (joelanford) [inline doc/user/metrics/service-monitor.md]:
```suggestion
# Using the ServiceMonitor prometheus-operator CRD with Operator SDK
```

File: README.md [hunk 10]
```diff
@@ -136,7 +136,7 @@ $ kubectl delete -f deploy/crds/app.example.com_appservices_crd.yaml
 
 To learn more about the SDK CLI, see the [SDK CLI Reference][sdk_cli_ref], or run `operator-sdk [command] -h`.
 
-## User Guides
+## User Guides for Operator SDK
```
Comment (joelanford) [inline README.md]: Revert. I think we're mostly worried about document titles, not section headers within the document.

```suggestion
## User Guides
```

File: doc/ansible/dev/advanced_options.md [hunk 11]
```diff
@@ -1,4 +1,4 @@
-## Advanced Options
+## Ansible Advanced Options for Operator SDK
```
Comment (joelanford) [inline doc/ansible/dev/advanced_options.md]:
```suggestion
# Advanced Options for Operator SDK Ansible-based Operators
```

File: doc/operator-scope.md [hunk 12]
```diff
@@ -1,4 +1,4 @@
-## Operator scope
+## Operator scope with Operator SDK
```
Comment (joelanford) [inline doc/operator-scope.md]:
```suggestion
# Operator SDK: Operator Scope
```

File: doc/ansible/dev/dependent_watches.md [hunk 13]
```diff
@@ -1,4 +1,4 @@
-# Dependent Watches
+# Ansible dependent Watches with Operator SDK
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply doc/ansible/dev/dependent_watches.md]: Done 👍

File: doc/ansible/dev/finalizers.md [hunk 14]
```diff
@@ -1,4 +1,4 @@
-# Handling deletion events
+# Ansible Handling deletion events with Operator SDK
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply doc/ansible/dev/finalizers.md]: Done 👍

File: doc/ansible/dev/testing_guide.md [hunk 15]
```diff
@@ -1,4 +1,4 @@
-# Testing Ansible Operators with Molecule
+# Ansible Testing Ansible based-Operators with Molecule for Operator SDK
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply doc/ansible/dev/testing_guide.md]: Done 👍

File: doc/ansible/project_layout.md [hunk 16]
```diff
@@ -1,4 +1,4 @@
-# Project Scaffolding Layout
+# Ansible Project Scaffolding Layout with Operator SDK
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply doc/ansible/project_layout.md]: Done 👍

File: doc/ansible/information-flow-ansible-operator.md [hunk 17]
```diff
@@ -1,4 +1,4 @@
-# Information Flow for Ansible Operator
+# Information Flow for Ansible based-operator with Operator SDK
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply doc/ansible/information-flow-ansible-operator.md]: done 👍

File: doc/user/metrics/README.md (hunk 8 above)
Comment (camilamacedo86) [reply doc/user/metrics/README.md]: Done 👍

File: doc/user/metrics/service-monitor.md (hunk 9 above)
Comment (camilamacedo86) [reply doc/user/metrics/service-monitor.md]: Done 👍

File: README.md (hunk 10 above)
Comment (camilamacedo86) [reply README.md]: done 👍

File: doc/ansible/dev/advanced_options.md (hunk 11 above)
Comment (camilamacedo86) [reply doc/ansible/dev/advanced_options.md]: done 👍

File: doc/operator-scope.md (hunk 12 above)
Comment (camilamacedo86) [reply doc/operator-scope.md]: Done 👍

**Comment by camilamacedo86** [comment]
Hi @joelanford, 

Thank your for your review. All requested changes are done 👍

File: doc/dev/release.md [hunk 18]
```diff
@@ -1,4 +1,4 @@
-# Releases
+# Releases for Operator SDK
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply doc/dev/release.md]: Done 👍

File: doc/dev/developer_guide.md [hunk 19]
```diff
@@ -1,4 +1,4 @@
-# Operator SDK Developer guide
+# Developer guide for Operator SDK
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply doc/dev/developer_guide.md]: Done 👍

File: doc/dev/reporting_bugs.md [hunk 20]
```diff
@@ -1,4 +1,4 @@
-# Reporting bugs
+# Reporting bugs for Operator SDK
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply doc/dev/reporting_bugs.md]: Done 👍

File: doc/dev/testing/running-the-tests.md [hunk 21]
```diff
@@ -1,4 +1,4 @@
-# Running the SDK Tests
+# Running the Tests with Operator SDK
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply doc/dev/testing/running-the-tests.md]: Done 👍

File: doc/faq.md [hunk 22]
```diff
@@ -1,4 +1,4 @@
-# FAQ
+# Operator SDK FAQ
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply doc/faq.md]: Done 👍

File: doc/helm/project_layout.md [hunk 23]
```diff
@@ -1,4 +1,4 @@
-# Project Scaffolding Layout
+# Helm Project Scaffolding Layout with Operator SDK
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply doc/helm/project_layout.md]: Done 👍

File: doc/migration/v0.1.0-migration-guide.md [hunk 24]
```diff
@@ -1,4 +1,4 @@
-# Migration Guide from v0.0.x to v0.1.0
+# Migration Guide from v0.0.x to v0.1.0 for Operator SDK
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply doc/migration/v0.1.0-migration-guide.md]: Done 👍

File: doc/migration/version-upgrade-guide.md [hunk 25]
```diff
@@ -1,4 +1,4 @@
-# Version upgrade guide
+# Version upgrade guide for Operator SDK
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply doc/migration/version-upgrade-guide.md]: Done 👍

File: doc/proposals/README.md (hunk 1 above)
Comment (camilamacedo86) [reply doc/proposals/README.md]: Done 👍

File: doc/user/event-filtering.md (hunk 6 above)
Comment (camilamacedo86) [reply doc/user/event-filtering.md]: done 👍

File: doc/user-guide.md (hunk 4 above)
Comment (camilamacedo86) [reply doc/user-guide.md]: Done 👍

File: doc/sdk-cli-reference.md (hunk 2 above)
Comment (camilamacedo86) [reply doc/sdk-cli-reference.md]: done 👍

File: doc/test-framework/scorecard.md (hunk 3 above)
Comment (camilamacedo86) [reply doc/test-framework/scorecard.md]: Done 👍

File: doc/user/client.md (hunk 5 above)
Comment (camilamacedo86) [reply doc/user/client.md]: Done 👍

File: doc/user/logging.md (hunk 7 above)
Comment (camilamacedo86) [reply doc/user/logging.md]: Done 👍

File: doc/ansible/dev/apb_migration_guide.md [hunk 26]
```diff
@@ -1,4 +1,4 @@
-# Migrating an Ansible Playbook Bundle (APB) to an Ansible-based Operator
+# Migrating an Ansible Playbook Bundle (APB) to an Ansible-based Operator for Operator SDK
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply doc/ansible/dev/apb_migration_guide.md]: Done 👍

File: doc/proposals/ansible-operator-status.md [hunk 27]
```diff
@@ -1,4 +1,4 @@
-# Ansible Operator Status Proposal
+# Ansible based-Operator Status Proposal for Operator SDK
```
Comment (joelanford) [inline doc/proposals/ansible-operator-status.md]:
```suggestion
# Ansible-based Operator Status Proposal for Operator SDK
```

Ditto for other places. It should be `Ansible-based` and `Helm-based`, not `based-operator`

File: doc/proposals/TEMPLATE.md [hunk 28]
```diff
@@ -1,4 +1,4 @@
-## <Title of Proposal>
+# <Title of Proposal> + Proposal for Operator SDK
```
Comment (joelanford) [inline doc/proposals/TEMPLATE.md]: I don't really think we should use the "Proposal for Operator SDK" suffix for the proposal titles. In fact, IIRC the proposals are the ones that were turning up in searches ahead of the actual documentation, so I'm not sure we want to make the proposals _more_ searchable.

Comment (camilamacedo86) [inline doc/proposals/TEMPLATE.md]:
```suggestion
# <Title of Proposal> 
```

**Pull request merged**